src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -203,17 +203,47 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
     });
   }
 
+  // Solution for checking partial element visibility adapted from
+  // https://www.codegrepper.com/code-examples/javascript/read+xlsx+and+xls+file+in+typescript
```

Comment:
The link is wrong? It brings me to a code snippet on opening excel files

Reply:
Thanks for pointing that out! I've fixed the link along with the errors for passing checks! :)